hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -1378,7 +1378,9 @@ public void joinCluster() throws IOException {
 
     // Start the chores
     master.getMasterProcedureExecutor().addChore(this.ritChore);
-    master.getMasterProcedureExecutor().addChore(this.deadMetricChore);
+    if (this.deadMetricChore != null) {
```

Comment:
nit: Guess this is redundant if the method accepts null values.

Reply:
Yes.

Reply:
I mean .. we can get rid of this if block? addChore has a similar check right?

Reply:
Yes, it can  be removed,  add it to show just where is wrong actually.

Reply:
nit: Remove the if condition here, since the method's args are Nullable?